public/styles/style.css

```diff
@@ -2,10 +2,12 @@
 @import url('https://fonts.googleapis.com/css2?family=Nanum+Myeongjo&display=swap');
 
 .container {
-  width: 900px;
+  width: 500px;
```

Comment:
are we using CSS modules? (this css file on its own makes me think you're not using Styled Components, which means you should be using CSS modules instead)